frontend/src/static/css/v_summary.scss

```diff
@@ -9,6 +9,8 @@
   .icon-button {
     color: mui-color('grey');
     cursor: pointer;
+    padding-left: 1.2px;
+    padding-right: 1.2px;
```

Comment:
can make this to one line padding: a b c d?

Reply:
Sure, no problem